coriolis_openstack_utils/cli/tenant.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2018 Cloudbase Solutions Srl
+# All Rights Reserved.
+
+from oslo_log import log as logging
+
+from cliff import lister
+
+from coriolis_openstack_utils import conf
+from coriolis_openstack_utils.actions import tenant_actions
+from coriolis_openstack_utils.cli import formatter
+
+
+LOG = logging.getLogger(__name__)
+
+
+class TenantMigrationFormatter(formatter.EntityFormatter):
+    columns = (
+        "Tenant Name",
+        "Tenant ID")
+
+    def _get_formatted_data(self, obj):
+        data = (
+            obj["name"],
+            obj["id"])
+
+        return data
+
+class MigrateTenant(lister.Lister):
+    def get_parser(self, prog_name):
+        parser = super(MigrateTenant, self).get_parser(prog_name)
+        source_group = parser.add_mutually_exclusive_group(required=True)
+        source_group.add_argument(
```

Comment:
In the long run, I think we should also add args like `--no-networks` or `--no-secgroups` etc..., but that would overcomplicate the command and/or config opts needed to migrate quite a bit (we'll need to rely on the user setting the right `network_map` values as `shared` networks on the destination so they can be automatically accessible to the tenant we just created)
Probably the best approach would be to allow the setting of these other flags iff `--no-instances` is also given.
Anyhow, don't worry about this now, only the above `--no-instances` suggestion.